examples/mlj/multiclass.jl

```diff
@@ -0,0 +1,31 @@
+module Multiclass
+
+using CatBoost.MLJCatBoostInterface
+using DataFrames
+using MLJBase
+using PythonCall
+
+# Initialize data
+train_data = DataFrame([coerce(["a", "a", "c"], MLJBase.Multiclass),
+                        coerce(["b", "b", "d"], MLJBase.Multiclass),
+                        coerce([0, 0, 1], OrderedFactor), [4, 5, 40], [5, 6, 50],
```

Comment:
Maybe for consistency make this `MLJBase.OrderedFactor`

Reply:
Done!